product_docs/docs/pem/9/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx

```diff
@@ -8,54 +8,83 @@ redirects:
   - /pem/latest/pem_online_help/09_toc_pem_configure_pgbouncer/02_pem_pgbouncer_preparing_dbserver/
 ---
 
-You must configure the PEM database server to work with PgBouncer. This example shows how to configure the PEM database server.
+You must configure dedicated users and create an SSL key and certificate on the PEM database server to enable connection pooling for PEM with PgBouncer. 
 
-1.  Create a dedicated user named pgbouncer on the PEM database server:
+This example shows how to prepare the PEM database serverwith the `enterprisedb` user on a RHEL-based operating system with EDB Postgres Advanced Server version 16. The location of your data, the configuration and key files, and the user you employ to perform the configuration are different depending on your OS and Postgres distribution. 
+
+## Prerequisites 
+
+-   You are connected to the `pem` database of the PEM database server. 
+
+-   You are connected as `enterprisedb` or `postgres` user. The user depends on your Postgres distribution.
+
+    | Postgres distribution        | User         |
+    |------------------------------|--------------|
+    | EDB Postgres Advanced Server | enterprisedb |
+    | EDB Postgres Extended Server | postgres     |
+    | PostgreSQL                   | postgres     |
+
+## Creating users and roles for PgBouncer-PEM connections
+
+1.  Create a dedicated user named pgbouncer with `pem_agent_pool` membership:
```

Comment:
Note what this user role is for.

Reply:
Since dave dropped the same comment below for all roles, commenting here: 

pem_agent_user1 ->  I think this is the proxy user that serves Agent-PgBouncer connections (with a single user).
pgbouncer -> I think this is the user that serves PgBouncer-PEM database connections (and running database queries). 
pem_admin1 -> I am not sure what this role does. Maybe since it has `pem_admin` and `pem_agent_pool` roles, it manages access and authenticatication for the PEM database? 

Just making guesses here to get us started, @edbjunwang or @edbshubham should we meet and discuss what this and the following roles do? or maybe you can provide a description per role?